pkg/flow/flow_updates_test.go

```diff
@@ -70,11 +70,8 @@ func TestController_Updates(t *testing.T) {
 	require.Equal(t, "10", in.(testcomponents.PassthroughConfig).Input)
 	require.Equal(t, "10", out.(testcomponents.PassthroughExports).Output)
 
-	in, out = getFields(t, ctrl.loader.Graph(), "testcomponents.summation.sum")
+	in, _ = getFields(t, ctrl.loader.Graph(), "testcomponents.summation.sum")
 	require.Equal(t, 10, in.(testcomponents.SummationConfig).Input)
-
-	// Since the lag is minimal, all updates will arrive to the final node.
```

Comment:
The lag is fine, but a pause on the machine can easily make the actual delay too high and an intermediate update will be skipped.